web-client/iron-remote-gui/index.html

```diff
@@ -1,20 +1,20 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
There is something weird happening here, is there something configured in your IDE that would be interfering with formatting?